src/fluree/crypto/secp256k1.cljc

```diff
@@ -59,74 +59,65 @@ public key, hex encoded."
   "Takes internal representation of a key-pair and returns X9.62 compressed encoded
   public key and private key as a map, with each value hex encoded."
   [pair]
-  (let [private #?(:clj (:private pair)
-                   :cljs (gobj/get pair "private"))
-        ^ECPoint public #?(:clj (:public pair)
-                           :cljs (gobj/get pair "public"))
-        x #?(:clj  (-> public .getAffineXCoord .toBigInteger
-                       (.toString 16) (encodings/pad-to-length 64))
+  (let [private         (:private pair)
+        ^ECPoint public (:public pair)
+        x #?(:clj       (-> public .getAffineXCoord .toBigInteger
+                            (.toString 16) (encodings/pad-to-length 64))
              :cljs (-> public (gobj/get "x") .toString
                        (.replace #"^0x" "") encodings/pad-hex))
-        y #?(:clj  (-> public .getAffineYCoord .toBigInteger
-                       (.toString 16))
+        y #?(:clj       (-> public .getAffineYCoord .toBigInteger
+                            (.toString 16))
              :cljs (-> public (gobj/get "y") .toString
-                       (.replace #"^0x" "") encodings/pad-hex))
-        pair-hex        {:private (encodings/biginteger->hex private)
-                         :public  (encodings/x962-encode x y)}]
-    #?(:clj  pair-hex
-       :cljs (clj->js pair-hex))))
+                       (.replace #"^0x" "") encodings/pad-hex))]
+    {:private (encodings/biginteger->hex private)
+     :public  (encodings/x962-encode x y)}))
+
 
 (defn public-key-from-private
   [private]
-  (let [private-bn #?(:clj  (cond
-                              (instance? BigInteger private) private
-                              (string? private) (BigInteger. ^String private 16))
-                      :cljs (-> (sjcl/bn. private)))]
+  (let [private-bn #?(:clj (cond
+                             (instance? BigInteger private) private
+                             (string? private) (BigInteger. ^String private 16))
+                      :cljs (sjcl/bn. private))]
     (when-not (valid-private? private-bn)
       (throw (ex-info "Invalid private key. Must be big integer and >= 1, <= curve modulus." {:private private})))
     #?(:clj  {:private private-bn
```

Comment:
Looks like there's an opportunity to push the reader conditional down to the :public keyword here.